cluster-autoscaler/test/e2e/main.go

```diff
@@ -0,0 +1,71 @@
+package main
```

Comment:
we might want to put this into upper level folder e.g `root/openshift/test/e2e` so it does not interfere with dep manager for `autoscaler/` binary

Reply:
I would even prefer to move it one dir up. Even though the repo carries more components, only the cluster-autoscaler is actually utilized.

Reply:
done, tests live in `test`